service/iot-service-samples/digitaltwin-service-samples/thermostat-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/Thermostat.java

```diff
@@ -170,7 +170,7 @@ private static void InvokeMethodOnRootLevel() throws IOException, InterruptedExc
                                 System.out.println("Invoked Command " + commandName + " failed: " + error);
                             }
                             else {
-                                System.out.println("Ensure the device sample is running for this sample to succeed - https://github.com/Azure/azure-iot-sdk-java/tree/master/device/iot-device-samples/pnp-device-sample/thermostat-device-sample.");
+                                System.out.println("Ensure the device sample is running for this sample to succeed - https://github.com/Azure/azure-iot-sdk-java/tree/mamainster/device/iot-device-samples/pnp-device-sample/thermostat-device-sample.");
```

Comment:
This got a bit messed up

Reply:
oops

Reply:
"Mainster" would be a cool default branch name, though